src/data/faq.json

```diff
@@ -101,6 +101,166 @@
+                    },
+                    {
+                        "title": "How is the proof of vaccination verified?",
+                        "anchor": "vaccination_certificate_verification",
+                        "active": true,
+                        "textblock": [
+                            "The vaccination certificate (QR code) is used digitally via the CovPass-App or the Corona-Warn-App (CWA) or alternatively as a machine-readable print. The vaccination certificate contains only information about the vaccination status, the name of the vaccinated person, and the date of birth. For service providers who want to check the vaccination status, there will be a verification app to check the vaccination certificate. This allows the vaccination status to be scanned in a similar way to a barcode of a flight or train ticket. Alternatively, proof with the analogue vaccination document is possible."
+                        ]
+                    },
+                    {
+                        "title": "Can you also store in the digital vaccination certificate that you have already been infected or tested negative?",
+                        "anchor": "vaccination_certificate_recovered_negative_test",
+                        "active": true,
+                        "textblock": [
+                            "Yes, negative tests or a recovery from the Corona virus can also be stored in the CovPass-App and Corona-Warn-App as a test certificate or recovery certificate. A recovery from a Corona infection is documented by means of a PCR test result."
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vaccination_certificate_eu",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "Does vaccination status affect the calculation of the risk? For example, if a vaccinated person has become infected and then warns other users?",
+                        "anchor": "vaccination_certificate_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "No, do not take into account in the risk calculation whether you are vaccinated against or recovered from a corona infection. The functions are independent of each other. The quality of your risk assessment does not change depending on whether you choose to save your vaccination certificate in the application, which underlines that this is completely voluntary. However, your personal vaccination status may affect your interpretation of the risk warning."

```

Comment:
"No, do not take into account" => "No, the vaccination status is not taken into account"

Reply:
@fynngodau There's one empty line too much in your commit suggestion. GitHub adds these, it's really weird.